src/main/java/org/wildfly/security/http/impl/UsernamePasswordAuthenticationMechanism.java

```diff
@@ -58,6 +59,12 @@ protected boolean authenticate(String realmName, String username, char[] passwor
         final PasswordGuessEvidence evidence = new PasswordGuessEvidence(password);
         EvidenceVerifyCallback evidenceVerifyCallback = new EvidenceVerifyCallback(evidence);
 
+        if (log.isDebugEnabled()) {
+           log.debug("Using UsernamePasswordAuthenticationMechanism for username authentication...");
```

Comment:
I would suggest for logging like this rather than logging three different messages it would be better to have a single log.debugf which contains the additional parameters in one.

In a server under heavy load the three related messages will be split up across the log file - elthough there are options to pull them back together it is best to not need to rely on them.